parts/linux/cloud-init/artifacts/nvidia-modprobe.service

```diff
@@ -5,5 +5,6 @@ Type=oneshot
 RemainAfterExit=true
 ExecStartPre=/bin/sh -c "dkms autoinstall --verbose"
 ExecStart=/bin/sh -c "nvidia-modprobe -u -c0"
+ExecStartPost=/bin/sh -c "sleep 10 && systemctl restart kubelet"
```

Comment:
aligns it with what was already inside cloud init, and was overriding this file in vhd